### PR TITLE
add examples using tidy eval and across()

### LINKED ISSUE #4792: Refresh `arrange()` docs
Pay careful attention to discussion of `desc()`

* [ ] Proofread title, description, and parameters 
* [ ] Proofread comments in examples 
* [ ] Ensure comment headings used to divide into scannable sections 
* [ ] Ensure there's one example that shows tidy eval and links to `?dplyr_tidy_eval` 
* [ ] Ensure there's one example that shows `across()` 
* [ ] Flag examples that seem overly complex for further review 
* [ ] Switch out mtcars/iris for something more interesting, or a smaller dataset constructed specifically to illustrate one point. 
* [ ] Review size of example outputs to ensure that you can easily see important rows and cols. 
* [ ] Datasets should always be tibbles to avoid use of `as_tibble()` distracting from the main point

### PR DESCRIPTION
**Pull request by shanepiesik**
Closes #4792

### CODE REVIEW

File: R/arrange.R [hunk 1]
```diff
@@ -49,6 +49,18 @@
 #' by_cyl %>% arrange(desc(wt))
 #' # Unless you specifically ask:
 #' by_cyl %>% arrange(desc(wt), .by_group = TRUE)
+#' # using Tidy Eval embracing
+#' tidy_eval_arrange <- function(data, arrange_var) {
```
Comment (romatik) [inline R/arrange.R]: Maybe use `.data` for consistency?

File: R/arrange.R [hunk 2]
```diff
@@ -49,6 +49,18 @@
...
+#'
+#'   arrange_var <- enquo(arrange_var)
+#'
+#'   data %>%
+#'     arrange(!! arrange_var)
```
Comment (romatik) [inline R/arrange.R]: Wouldn't embrace just be `arrange({{arrange_var}})` without line 55?

File: R/arrange.R (hunk 1 above)
Comment (shanepiesik) [reply R/arrange.R]: Yes, good point.

File: R/arrange.R (hunk 2 above)
Comment (shanepiesik) [reply R/arrange.R]: Yes, I went back and forth on using embracing or not, which resulted in this mix up between code and comments. I will change the code to use `{{ arrange_var }}`

**Comment by hadley** [comment]
Thanks @shanepiesik! (sorry it took so long to merge; I just got back from vacation)

**Pull request merged**